Tests.Integration.Presentation.Web/Users/UserTest.cs

```diff
@@ -119,6 +121,66 @@ public async Task Cannot_Get_Users_Where_User_Has_StakeHolder_Or_ApiAccess_If_No
             Assert.Equal(HttpStatusCode.Forbidden, result.StatusCode);
         }
 
+        [Fact]
+        public async Task Delete_User()
+        {
+            var userRole = OrganizationRole.LocalAdmin;
+
+            var (_, userId, organization) = await CreatePrerequisitesAsync(userRole);
+            var name = A<string>();
+
+            var project = await ItProjectHelper.CreateProject(name, organization.Id);
+            var dpr = await DataProcessingRegistrationHelper.CreateAsync(organization.Id, name);
+
+            await RightsHelper.AddUserRole(userId, organization.Id, RightsType.ItContractRights, name);
+            await RightsHelper.AddUserRole(userId, organization.Id, RightsType.OrganizationUnitRights, name);
+            await RightsHelper.AddUserRole(userId, organization.Id, RightsType.ItProjectRights, name, project.Id);
+            await RightsHelper.AddUserRole(userId, organization.Id, RightsType.ItSystemRights, name);
+            await RightsHelper.AddUserRole(userId, organization.Id, RightsType.DprRights, name, dpr.Id);
+
+            await ItProjectHelper.AddAssignmentAsync(organization.Id, userId, project.Id);
+            await ItProjectHelper.AddCommunicationAsync(organization.Id, userId, project.Id);
+            await ItProjectHelper.AddRiskAsync(organization.Id, userId, project.Id);
+            await ItProjectHelper.AddHandoverResponsibleAsync(project.Id, userId);
+            RightsHelper.AddSsoIdentityToUser(userId);
+
+            var deleteResponse = await UserHelper.SendDeleteUserAsync(userId);
+            Assert.Equal(HttpStatusCode.OK, deleteResponse.StatusCode);
+            
+            var getDeletedUserResponse = await UserHelper.GetUserByIdAsync(userId);
+
+            Assert.True(getDeletedUserResponse.Deleted);
+            Assert.False(getDeletedUserResponse.IsGlobalAdmin);
+            Assert.False(getDeletedUserResponse.HasApiAccess);
+            Assert.False(getDeletedUserResponse.HasStakeHolderAccess);
+
+            Assert.Contains("_deleted_user@kitos.dk", getDeletedUserResponse.Email);
+            Assert.Contains("(SLETTET)", getDeletedUserResponse.LastName);
+            Assert.NotNull(getDeletedUserResponse.EmailBeforeDeletion);
```

Comment:
Please verify that the email before delettion is actually the email before deletion